PigLatin/PigLatin.cs

```diff
@@ -6,16 +6,55 @@ class Program
     {
         public static void Main()
         {
-            // your code goes here
-
-            // leave this command at the end so your program does not close automatically
-            Console.ReadLine();
+            // Console.WriteLine("Enter a word: ");
+            // // leave this command at the end so your program does not close automatically
+            // String inWord = Console.ReadLine();
+            // Console.WriteLine("Your word became " + TranslateWord(inWord));
+            if(tests()){
+                Console.WriteLine("Tests passed.");
+            } else {
+                Console.WriteLine("Tests failed.");
+            }
+        }
+        public static bool tests(){
+            return 
+                TranslateWord("elephant") == "elephantyay" &&
+                TranslateWord("fox") == "oxfay" &&
+                TranslateWord("choice") == "oicechay" && 
+                TranslateWord("dye") == "yeday" && 
+                TranslateWord("bystander") == "ystanderbay" &&
+                TranslateWord("yellow") == "ellowyay" &&
+                TranslateWord("tsktsk") == "tsktskyay"; //your code was "tsktskay" instead of "tsktskyay"
```

Comment:
I was expecting it to be tsktskay, since there is no vowel. but this is fine since you made the comment.